paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -442,6 +443,25 @@ public class CoreOptions implements Serializable {
                             "If set to true, compactions and snapshot expiration will be skipped. "
                                     + "This option is used along with dedicated compact jobs.");
 
+    public static final ConfigOption<String> WRITE_SKIP_ACTIONS =
+            key("write.skip-actions")
+                    .stringType()
```

Comment:
enumType?

Reply:
This cannot be an enumType because this is a multi-option.